marlowe-playground-client/src/SimulationPage/View.purs

```diff
@@ -403,7 +404,9 @@ inputItem metadata _ person (ChoiceInput choiceId@(ChoiceId choiceName choiceOwn
     ( [ div [ classes [ ClassName "action-label" ] ]
           ( [ div [ class_ (ClassName "choice-input") ]
                 [ span [ class_ (ClassName "break-word-span") ] [ text "Choice ", b_ [ text (show choiceName <> ": ") ] ]
-                , marloweActionInput (SetChoice choiceId) chosenNum
+                , case mChoiceInfo of
+                    Just { choiceFormat: DecimalFormat numDecimals currencyLabel } -> marloweCurrencyInput (SetChoice choiceId) currencyLabel numDecimals chosenNum
+                    _ -> marloweCurrencyInput (SetChoice choiceId) "" 0 chosenNum
```

Comment:
I assume by a quick search that this is the only place where `marloweCurrencyInput` is used. I'd expect that `choice-input` has `flex` in it (or you should put a `flex` class next to it). 

I think it's better to add the grow/shrink parameters when you place `marloweCurrencyInput`, because in a different layout you might need to add this input in a grid, and then the hardcoded params for flex would not work. If you don't want for `marloweCurrencyInput` to accept class parameters, then making it `w-full` and adding a div wrapper to limit the width would do the trick